### PR TITLE
Improve monetization UI

### PR DESCRIPTION
**Pull request by shilmyhasan**
The PR addresses the improvements and issues raised in https://github.com/wso2/product-apim/issues/6762

image

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx [hunk 1]
```diff
@@ -80,126 +105,102 @@ class BusinessPlans extends Component {
...
-                                        defaultMessage='Unchecked polices are not monetized, click `Save` to monetize'
-                                    />
-                                </Typography>
-                            </Grid>
+                            <Banner
+                                disableActions
+                                dense
+                                paperProps={{ elevation: 1 }}
+                                type='info'
+                                message='Click Save to monetize all unmonetized policies'
```
Comment (chanaka3d) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx]: Need i18n

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx [hunk 2]
```diff
@@ -80,126 +105,102 @@ class BusinessPlans extends Component {
...
-                                        defaultMessage='No commercial policies to monetize'
-                                    />
-                                </Typography>
-                            </Grid>
+                            <Banner
+                                disableActions
+                                dense
+                                paperProps={{ elevation: 1 }}
+                                type='info'
+                                message='No commercial policies to monetize'
```
Comment (chanaka3d) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx]: Need i18n

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/index.jsx [hunk 3]
```diff
@@ -207,76 +208,102 @@ class Monetization extends Component {
             return <Progress />;
         }
         return (
-            <Grid item xs={6}>
-                <Typography variant='h4' gutterBottom>
-                    <FormattedMessage id='Apis.Details.Monetization.Index.monetization' defaultMessage='Monetization' />
-                </Typography>
+            <Grid>
```
Comment (chanaka3d) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/index.jsx]: You can get rid of this tag.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx (hunk 1 above)
Comment (shilmyhasan) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx]: The support fot i18n is not yet available in above Banner component at [Banner](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Shared/Banner.jsx#L65)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx (hunk 2 above)
Comment (shilmyhasan) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx]: The support fot i18n is not yet available in above Banner component at [Banner](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Shared/Banner.jsx#L65)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/index.jsx (hunk 3 above)
Comment (shilmyhasan) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/index.jsx]: fixed in d1ec43ed4f29d7fc678637fc70f1a68f9e8afca9

**Pull request merged**